via/static/scripts/video_player/components/Transcript.tsx

```diff
@@ -128,20 +124,37 @@ function TranscriptSegment({
 
   return (
     <li
-      className={classnames('flex flex-row p-1 hover:text-black', {
-        'bg-grey-2': isCurrent,
-        'text-grey-6': !isCurrent,
-        hidden,
-      })}
+      className={classnames(
+        'flex gap-x-3 p-1.5',
+        // Various tranparencies of a dark color are used for backgrounds to
+        // avoid obscuring scroll-hint shadows
+        'border-y hover:bg-slate-9/[.08]',
+        {
+          'bg-slate-9/[.08] border-grey-4/50 shadow-inner': isCurrent,
+          // Non-current entries have a bottom border only (subtle dotted)
+          'odd:bg-slate-9/[.03] border-t-transparent border-b-grey-2 border-b-dotted':
+            !isCurrent,
+          hidden,
+        }
+      )}
       data-is-current={isCurrent}
       data-testid="segment"
     >
       <button
         aria-label={timestamp}
         onClick={onSelect}
         className={classnames(
-          'pr-2 mr-3 hover:underline',
-
+          // TODO: Use shared Button to get these styles for free
+          'flex transition-colors focus-visible-ring rounded-sm',
+          // `peer` allows Tailwind styling based on sibling state
+          'peer',
+          'font-medium hover:underline',
+          {
+            // Colors are one tick lighter than segment text
+            'text-stone-700': isCurrent,
+            'text-stone-500': !isCurrent,
+            'hover:text-stone-800': true,
```

Comment:
Why adding this here with `true` instead of adding it as a plain string before the object itself?